topics/statistics/tutorials/RNN/tutorial.md

```diff
@@ -211,28 +211,53 @@ cost function via backpropagation.
 
 # Get data
 
+Let's start with a fresh history.
+
+> ### {% icon hands_on %} Hands-on: Create history
+>
+> 1. Make sure you have an empty analysis history.
+>
+>    {% snippet snippets/create_new_history.md %}
+>
+> 2. **Rename your history** to make it easy to recognize
+>
+>    > ### {% icon tip %} Rename a history
+>    >
+>    > * Click on the title of the history (by default the title is `Unnamed history`)
+>    >
+>    >   ![Renaming history](../../../../shared/images/rename_history.png)
+>    >
+>    > * Type `Galaxy Introduction` as the name
+>    > * Press <kbd>Enter</kbd>
+>    >
+>    {: .tip}
+>
+{: .hands_on}
+
+## Data upload
+
 > ### {% icon hands_on %} Hands-on: Data upload
 >
-> - {% tool [Import some data](upload1) %}
->    - Create a new history for this tutorial
->    - Import the files from [Zenodo](https://zenodo.org/record/4477881) or from the shared data library
+> 1. Create a new history for this tutorial
```

Comment:
a new history was already created in the step before? Maybe `1. Create a new history for this tutorial` is not necessary. Or it can be rephrased it like `<1. Create a new history for this tutorial if not created in the first step`

Reply:
@anuprulez which first step are you referring to? Thanks

Reply:
the one I commented on above, I think.

Reply:
@kxk302 `Get data` section recommends creating a new history and the `Data upload` section also recommends to create a new history. Should we merge these into one block?

Reply:
Aah, I see it now :) I'll merge them. Thx.